### PR TITLE
snek? snek?! SNEEEKEKKKKKK!!!!!

### PR DESCRIPTION
**Pull request by Aurorablade**
Adds stealth botany from this PR https://github.com/tgstation/tgstation/pull/18741

I have messed up the alt appearance code somehow so eh...

Also tempted tro add some potted plants to the new ops base

:cl: Fethas
rscadd: you can now pick up potted plants and stealth
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
and here I thought you meant something from actual Hydroponics Botany.

I don't know, seems a little too OP for hiding purposes. I can't say though

**Comment by Ar3nn** [comment]
So wait you pick up a plant and it hides you?

**Comment by Fox-McCloud** [comment]
Need to hit the away missions, as well, @Aurorablade

**Comment by Aurorablade** [comment]
@Fox-McCloud gotcha, also can anyone see whats up with the alt apperance code?

**Comment by FreeStylaLT** [comment]
for once

i will be groot

**Comment by Fox-McCloud** [comment]
>i will be groot

*glares at*

**Comment by Earthdivine** [comment]
>I will be groot

I am having second thoughts of this

**Comment by KasparoVy** [comment]
Dude, this is bloody hilarious. Actually excited to see it in action.

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -191,20 +191,34 @@
...
 
-/obj/structure/flora/kirbyplants/New()
+/obj/item/weapon/twohanded/required/kirbyplants/New()
 	..()
 	icon_state = "plant-[rand(1,35)]"
 	if(prob(1))
 		icon_state = "plant-36"
 
-/obj/structure/flora/kirbyplants/dead
+/obj/item/weapon/twohanded/flora/kirbyplants/equipped(mob/living/user)
```
Comment (tigercat2000) [inline code/game/objects/structures/flora.dm]: this is `flora` instead of `required`

**Comment by Fox-McCloud** [comment]
This ready to go @Aurorablade?

**Pull request merged**